package.json

```diff
@@ -12,13 +12,6 @@
     "lint-staged"
   ],
   "lint-staged": {
```

Comment:
Replace your package.json with the org repo's copy and manually run `npm run lint` before you push up your responses to my comments please. 
https://github.com/hrla14-snippr/snippr/blob/master/package.json